### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
- [x] Fixed negative values in selector names.
- [x] Extract two classes from the palettes mixin (to avoid multiple instantiations).
- [x] Replace prefix by its variable.

### CODE REVIEW

**Review by BertrandPodevin — DISMISSED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1728

File: packages/scss/src/commons/core.scss [hunk 1]
```diff
@@ -115,14 +115,6 @@ $states: 'error,' 'warning', 'success';
 			color: var(--palettes-#{$palette}-700) !important;
```
[the remark this answers is not in the document]
Comment (vvalentin-lucca) [reply packages/scss/src/commons/core.scss:115]: Bien vu !

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1728

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-120-166ffbc0-b8db-47d8-9248-0e360d417992/ui-diff.html)

**Review by BertrandPodevin — APPROVED** [review]

**Pull request merged**